src/js/components/todo/todoList.js

```diff
@@ -0,0 +1,80 @@
+import { $, $closet, $closetAttr, TODO_SELCTOR } from "../../utils/dom.js";
+import {
+  checkClassName,
+  checkKey,
+  checkLocalName,
+  getValue,
+  isEmptyValue,
+  setClass,
+} from "../../utils/eventUtils.js";
+import { ILLEGAL_MESSAGE } from "../../utils/Message.js";
+import { todoItemTemplate } from "./todoItem.js";
+
+export default function TodoList(app) {
+  const $todoList = $(TODO_SELCTOR.TODO_LIST_CONTAINER);
+
+  this.render = (members) => {
+    members.forEach((member) => {
+      const $todoMember = $(TODO_SELCTOR.CONTAINER(member.getId()), $todoList);
+      const template = app
+        .checkStatus(member.getId(), member.getTodoList())
+        .map((item) => todoItemTemplate(item));
+      $(TODO_SELCTOR.TODO_LIST, $todoMember).innerHTML = template.join("\n");
+    });
+  };
+
+  const editing = (event) => {
+    const target = $closet(event.target, TODO_SELCTOR.TODO_ID[0]);
+    target && setClass(target, [TODO_SELCTOR.EDIT]);
+  };
+
+  const onClickHandler = (event) => {
+    const target = event.target;
+    if (checkClassName(event, "toggle")) {
```

Comment:
핸들러 함수에서 이벤트 객체의 target 만 쓰실 거면 아예 아래처럼 파라미터로 가져오셔도 돼요ㅎㅎ
```suggestion
  const onClickHandler = ({target}) => {
    if (checkClassName(event, "toggle")) {
```